ocean-client/src/vault-maxi.ts

```diff
@@ -26,10 +26,11 @@ class maxiEvent {
 export async function main(event:maxiEvent): Promise<Object> {
     let settings = await new Store().fetchSettings()
 
-    if(event) {
-        console.log("received event "+JSON.stringify(event))
+    const telegram = new Telegram(settings, "[Maxi " + (settings.vault?.length > 6 ? settings.vault.substring(0, 6) : "...") + "]")
+    if (event) {
+        console.log("received event " + JSON.stringify(event))
         if (event.overrideSettings) {
-            if(event.overrideSettings.maxCollateralRatio)
+            if (event.overrideSettings.maxCollateralRatio)
                 settings.maxCollateralRatio= event.overrideSettings.maxCollateralRatio
             if(event.overrideSettings.minCollateralRatio)
```

Comment:
minor: doesn't need to be changed, but above you added the whitespace and here it got missed